openapi/azure.openai.embeddings/Ballerina.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+
+license = ["Apache-2.0"]
+keywords = ["AI/Azure", "AI/OpenAI", "Cost/Paid", "OpenAI embeddings", "GPT-3 embeddings" ,"Vendor/Microsoft"]
```

Comment:
Here for the keywords can you check with @isuruRuhu and have a standard way?

Reply:
Resolved in be0689529949000148dc55e5c896a8a376f97565